packages/gtk3.rb

```diff
@@ -49,26 +51,33 @@ def self.build
             builddir"
     system 'meson configure builddir'
     system 'ninja -C builddir'
-    @file = <<~GTK3_CONFIG_HEREDOC
-      [Settings]
-      gtk-application-prefer-dark-theme = false
-      gtk-icon-theme-name = hicolor
-      gtk-fallback-icon-theme = gnome
-      gtk-font-name = Arial 10
-    GTK3_CONFIG_HEREDOC
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
     system "sed -i 's,null,,g'  #{CREW_DEST_LIB_PREFIX}/pkgconfig/gtk*.pc"
     FileUtils.mkdir_p("#{CREW_DEST_HOME}/.config/gtk-3.0")
-    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @file)
+    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @gtk3settings)
```

Comment:
Where is `@gtk3settings` defined?  I see it in `postinstall` but doesn't that come after `install`?

Reply:
good catch, that's extraneous.